pkgs/desktops/gnome/extensions/taskwhisperer/fix-paths.patch

```diff
@@ -1,99 +1,99 @@
-             let shellProcExited = true;
-             shellProc.wait_finish(result);
-@@ -344,7 +344,7 @@ const TaskService = class TaskService {
-         if (!taskID) {
-             return;
-         }
--        let shellProc = Gio.Subprocess.new(['task', taskID.toString(), 'stop'], Gio.SubprocessFlags.STDOUT_PIPE);
-+        let shellProc = Gio.Subprocess.new(['@task@', taskID.toString(), 'stop'], Gio.SubprocessFlags.STDOUT_PIPE);
-         shellProc.wait_async(null, function (obj, result) {
-             let shellProcExited = true;
-             shellProc.wait_finish(result);
-@@ -374,7 +374,7 @@ const TaskService = class TaskService {
-         // FIXME: Gio.Subprocess: due to only passing string vector is allowed, it's not possible to directly pass the
-         //        input of the user to subprocess (why & how, if you can answer then please send msg to fh@infinicode.de)
-         //        bypassing problem with own shell script
--        let shellProc = Gio.Subprocess.new(['/bin/sh', EXTENSIONDIR + '/extra/modify.sh', taskID.toString(), params], Gio.SubprocessFlags.STDOUT_PIPE + Gio.SubprocessFlags.STDERR_MERGE);
-+        let shellProc = Gio.Subprocess.new(['@shell@', EXTENSIONDIR + '/extra/modify.sh', taskID.toString(), params], Gio.SubprocessFlags.STDOUT_PIPE + Gio.SubprocessFlags.STDERR_MERGE);
- 
-         shellProc.wait_async(null, function (obj, result) {
-             let shellProcExited = true;
-@@ -403,7 +403,7 @@ const TaskService = class TaskService {
-         // FIXME: Gio.Subprocess: due to only passing string vector is allowed, it's not possible to directly pass the
-         //        input of the user to subprocess (why & how, if you can answer then please send msg to fh@infinicode.de)
-         //        bypassing problem with own shell script
--        let shellProc = Gio.Subprocess.new(['/bin/sh', EXTENSIONDIR + '/extra/create.sh', params], Gio.SubprocessFlags.STDOUT_PIPE + Gio.SubprocessFlags.STDERR_MERGE);
-+        let shellProc = Gio.Subprocess.new(['@shell@', EXTENSIONDIR + '/extra/create.sh', params], Gio.SubprocessFlags.STDOUT_PIPE + Gio.SubprocessFlags.STDERR_MERGE);
- 
-         shellProc.wait_async(null, function (obj, result) {
-             let shellProcExited = true;
-@@ -432,7 +432,7 @@ const TaskService = class TaskService {
-         let shellProc;
- 
-         try {
--            shellProc = Gio.Subprocess.new(['task', 'sync'], Gio.SubprocessFlags.STDOUT_PIPE);
-+            shellProc = Gio.Subprocess.new(['@task@', 'sync'], Gio.SubprocessFlags.STDOUT_PIPE);
-         } catch (err) {
-             onError(err);
-             return;
+diff --git a/taskwhisperer-extension@infinicode.de/metadata.json b/taskwhisperer-extension@infinicode.de/metadata.json
+index 2f1471c..a84bdf4 100644
+--- a/taskwhisperer-extension@infinicode.de/metadata.json
++++ b/taskwhisperer-extension@infinicode.de/metadata.json
+@@ -6,7 +6,8 @@
+     "3.32",
+     "3.36",
+     "3.38",
+-    "40"
++    "40",
++    "41"
```

Comment:
This allows us to run the extension on the latest gnome desktop in nixpkgs, I have tested it and it works